### ISSUE TITLE
Train the model

### ISSUE THREAD

**Issue by arlenecse** [amaccodes/hm-fashion-recommender#4]
Is not dependent on the completion of the other tasks

- [ ] Initialize the model
- [ ] Loss function
- [ ] Optimizer
- [ ] Batch size
- [ ] Epochs
- [ ] Model prediction for each batch
